extendable_fastapi/fastapi_dispatcher.py

```diff
@@ -3,15 +3,16 @@
 
 from contextlib import contextmanager
 
+from odoo.http import _dispatchers
+
 from odoo.addons.extendable.registry import _extendable_registries_database
-from odoo.addons.fastapi.fastapi_dispatcher import (
-    FastApiDispatcher as BaseFastApiDispatcher,
-)
 
 from extendable import context
 
 
-class FastApiDispatcher(BaseFastApiDispatcher):
+# Inherit from last dispatcher of fastapi registered
+# This handles multiple overloaded dispatchers
+class FastApiDispatcher(_dispatchers["fastapi"]):
```

Comment:
This approach doesn't take into account that the module where your dispatcher is defined is installed or not into the database...